release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -629,6 +636,20 @@ For further information about enabling more control over the C-states, see xref:
 ==== Support for provisioning IPv6 spoke clusters from dual-stack hub clusters
 With this update, you can provision IPv6 address spoke clusters from dual-stack hub clusters. In a zero touch provisioning (ZTP) environment, the HTTP server on the hub cluster that hosts the boot ISO now listens on both IPv4 and IPv6 networks. The provisioning service also checks the baseboard management controller (BMC) address scheme on the target spoke cluster and provides a matching URL for the installation media. These updates offer the ability to provision single-stack, IPv6 spoke clusters from a dual-stack hub cluster.
 
+[id="ocp-4-14-nw-shiftstack-dual-stack"]
+==== Support for dual-stack networking for {rh-openstack} clusters (Technology Preview)
+
+Dual-stack network configuration is now available for clusters that run on {rh-openstack}. You can configure dual-stack networking during the deployment of a cluster on installer-provisioned infrastructure.
+
+[id="ocp-4-14-nw-shiftstack-manage-security-groups"]
+==== Security group management for {rh-openstack} clusters
```

Comment:
might be worth to point to the feature docs, in case there is any

Reply:
There are some instructions in the installation assemblies, but I'm not sure how useful dropping someone partway into an installation process is.